easysocial/libraries/mappingHelper.php

```diff
@@ -1819,4 +1921,59 @@ public function videoMap($rows)
 			return $result;
 		}
 	}
+
+	/**
+	 * Function for getting avtar image
+	 *
+	 * @param   array  $row   array of data
+	 * @param   array  $item  array of data
+	 * 
+	 * @return  array
```

Comment:
@Pratikshakhandagale  - update the comment as the function is not returning anything.